src/include/storage/store/string_column_chunk.h

```diff
@@ -31,11 +31,11 @@ class StringColumnChunk : public ColumnChunk {
 
     uint64_t getStringLength(common::offset_t pos) const {
         auto index = ColumnChunk::getValue<DictionaryChunk::string_index_t>(pos);
-        return dictionaryChunk.getStringLength(index);
+        return dictionaryChunk->getStringLength(index);
     }
 
-    inline DictionaryChunk& getDictionaryChunk() { return dictionaryChunk; }
-    inline const DictionaryChunk& getDictionaryChunk() const { return dictionaryChunk; }
+    inline DictionaryChunk& getDictionaryChunk() { return *dictionaryChunk; }
```

Comment:
I'm thinking to form a convention of naming functions returning non-const reference with suffix `UnSafe` (or maybe other suffix to be more explicit). What do you think? Is this necessary?